### ISSUE TITLE
Updates to Vault.md for UI changes

### ISSUE THREAD

**Issue by britttttk** [btcpayserver/btcpayserver-doc#590]
When this question is asked in the chat, "Do I need to use Vault" or similar, it's hard to answer without more information. I think that we should add a Vault question to Wallet FAQ for users who are not sure what it is from seeing it in the BTCPay UI.

I think [this Vault Blog post](https://blog.btcpayserver.org/btcpay-vault/) might be good for this since it is a broad explanation and has links to lots of docs references.

**Comment by pavlenex** [comment]
Hijacking the issue a bit but, problem with Vault is people don't have to know about it nor they should. In hindsight, we should have pushed with "hardware wallet integration" terminology. Download an app (vault) is the most attention the vault should have gotten.

We don't mention vault in the UI anymore, it's now hardware wallet, which is clear from the start.

![HW](https://user-images.githubusercontent.com/36959754/84980924-a0f3c480-b133-11ea-9342-c6d1818d122d.PNG)

My suggestion is we discontinue the vault term entirely and push with "HWI", we're already doing it in the structure as well. 
![HWI](https://user-images.githubusercontent.com/36959754/84980666-f1b6ed80-b132-11ea-9735-c4d240d11b00.PNG)

As for this issue, I guess, after restructure of Vault.md and related docs, we can have "Can I use my hardware wallet with BTCPay Server" type of questions.

**Comment by britttttk** [comment]
@pavlenex Ok that makes sense, since it's not listed as Vault in the import/sign dropdowns anymore and there is `Wallet.md#can-i-use-a-hardware-wallet-with-btcpay-server` which can help users discover Vault docs. 

Changing this issue to replace the 2 images and text which point the user to choose Vault (preview) from dropdown. The video mentions they may change and refer to docs so I guess we should make those changes here.

**Comment by pavlenex** [comment]
Awesome I'll open a new issue after we tackle this that will tackle the Vault.md itself change the name to HardwareWalletIntegration, and few more things in docs, so it's less "vault and more "hardware wallet integration" We'd have to technically dumb it down, but whoever wants to, can go to vault on github to learn more.
